app/models/internship.rb

```diff
@@ -10,5 +10,6 @@ class Internship < ActiveRecord::Base
   validates :company_id, presence: true, uniqueness: { scope: :cohort_id }
 
   delegate :name, to: :company, prefix: :company
-  
+  default_scope { joins(:company).order("name") }
```

Comment:
Not a good idea to have a join in the default scope. This will slow down every internship query - at some point, maybe somebody wants internships without companies, and doesn't realize there is a default scope on. Use a named scope instead.